.circleci/config.yml

```diff
@@ -75,3 +75,53 @@ jobs:
             export PYTHONPATH=${PYTHONPATH}:$(pwd)/lib && source venv/bin/activate
             python3 mrobpy/tests/PC_align.py
           when: always
+  build-wheel:
+    docker:
+      - image: miloserdoval/building-wheel-manylinux
```

Comment:
is there a way to remove any dependency from external accounts. I think for now it is fine, but in the long term we would like to have an account belonging to the lab or something like that. What do you think?

Reply:
One of our next tasks is to abandon this docker container by changing CMake. Then the manylinux container (https://github.com/pypa/manylinux) will be used.

Reply:
ok, so this version is temporary but still, you want to merge it to master?
We can do that, although let me know what are the next plans because we would like to change some things before doing the first tag (I will write the question).